src/include/storage/store/table.h

```diff
@@ -22,6 +22,7 @@ struct TableScanState {
     std::unique_ptr<TableDataScanState> dataScanState;
     common::node_group_idx_t nodeGroupIdx = common::INVALID_NODE_GROUP_IDX;
 
+    // Zone map
```

Comment:
Either remove the comment or be more explicit to explain the usage of `columnPredicateSets` and `zoneMapResult`.